extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -317,6 +317,13 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
             };
           });
       sandbox.stub(impl, 'getAmpDoc', () => {return document;});
+      // Reproduced from noopMethods in ads/google/a4a/test/test-utils.js,
+      // to fix failures when this is run after 'gulp build', without a 'dist'.
+      sandbox.stub(impl, 'getPageLayoutBox', () => {
+        return {
+          top: 11, left: 12, right: 0, bottom: 0, width: 0, height: 0,
```

Comment:
'gulp dist' does a more comprehensive build, including creating and serving a minified version of the AMP runtime. This shouldn't be necessary for running unit tests, and results in a lot of wasted CPU time on Travis when multiplied across all the commits across all the PRs pushed by people on the AMP team. I'd imagine that you can get your tests to pass without this stub with a little refactoring / reuse of the code in test-utils.js.

Thanks for the review.